src/analysis/PlutoReactiveCore/ExpressionExplorer.jl

```diff
@@ -78,17 +91,39 @@ function macro_has_special_heuristic_inside(; symstate::SymbolsState, expr::Expr
     # Also, because I'm lazy and don't want to copy any code, imma use cell_precedence_heuristic here.
     # Sad part is, that this will also include other symbols used in this macro... but come'on
     node = ReactiveNode(symstate)
-    code = Pluto.ExprAnalysisCache(
+    code = PlutoReactiveCore.ExprAnalysisCache(
         parsedcode = expr,
         module_usings_imports = ExpressionExplorer.compute_usings_imports(expr),
     )
 
-    return Pluto.cell_precedence_heuristic(node, code) < Pluto.DEFAULT_PRECEDENCE_HEURISTIC
+    return PlutoReactiveCore.cell_precedence_heuristic(node, code) < PlutoReactiveCore.DEFAULT_PRECEDENCE_HEURISTIC
 end
 
 const can_macroexpand_no_bind = Set(Symbol.(["@md_str", "Markdown.@md_str", "@gensym", "Base.@gensym", "@enum", "Base.@enum", "@assert", "Base.@assert", "@cmd"]))
 const can_macroexpand = can_macroexpand_no_bind ∪ Set(Symbol.(["@bind", "PlutoRunner.@bind"]))
 
+const plutorunner_id = Base.PkgId(Base.UUID("dc6b355a-2368-4481-ae6d-ae0351418d79"), "PlutoRunner")
+const pluto_id = Base.PkgId(Base.UUID("c3e4b0f8-55cb-11ea-2926-15256bba5781"), "Pluto")
+const found_plutorunner = Ref{Union{Nothing,Module}}(nothing)
+
+function get_plutorunner()
+    fpr = found_plutorunner[]
+    if fpr === nothing
+        # lets try really hard to find it!
+        if haskey(Base.loaded_modules, plutorunner_id)
+            found_plutorunner[] = Base.loaded_modules[plutorunner_id]
+        elseif haskey(Base.loaded_modules, pluto_id)
+            found_plutorunner[] = Base.loaded_modules[pluto_id].PlutoRunner
```

Comment:
I don't think we want this one first. If Pluto is loaded in the process then Pluto.PlutoRunner should be preferred (supposing that PlutoReactiveCore is most likely being called by Pluto).
```suggestion
        if haskey(Base.loaded_modules, pluto_id)
            found_plutorunner[] = Base.loaded_modules[pluto_id].PlutoRunner
        elseif haskey(Base.loaded_modules, plutorunner_id)
            found_plutorunner[] = Base.loaded_modules[plutorunner_id]
```